packages/salesforcedx-vscode-apex-debugger/package.json

```diff
@@ -24,17 +24,17 @@
     "Debuggers"
   ],
   "dependencies": {
-    "@salesforce/salesforcedx-apex-debugger": "56.1.0",
+    "@salesforce/salesforcedx-apex-debugger": "56.2.0",
     "vscode-debugprotocol": "1.28.0",
     "vscode-extension-telemetry": "0.0.17"
   },
   "devDependencies": {
-    "@salesforce/salesforcedx-test-utils-vscode": "56.1.0",
+    "@salesforce/salesforcedx-test-utils-vscode": "56.2.0",
     "@types/chai": "4.3.3",
     "@types/mocha": "^5",
     "@types/node": "12.0.12",
     "@types/sinon": "^2.3.7",
-    "@types/vscode": "1.49.0",
+    "@types/vscode": "1.72.0",
```

Comment:
Note I bumped the version of vscode to 1.72. This is the version released in 10/2021.  Verified via appsights that our user base is keeping vscode up to date.